src/main.rs

```diff
@@ -87,6 +89,11 @@ fn do_clone_and_checkout(
     Ok(git_rev)
 }
 
+fn do_remove_folder(path: &str) -> Result<(), RemoveFolderError> {
+    fs::remove_dir_all(path)
+        .map_err(|_| RemoveFolderError(format!("Could not delete folder: {}", path)))
```

Comment:
Does this yield an error if the build folder is not present? Should probably not do that?

Reply:
that is indeed a bug, fixed ist last commit